.rultor.yml

```diff
@@ -15,4 +19,4 @@ release:
   script: bash .misc/release.sh
 
 decrypt:
-  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
+  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"  
```

Comment:
trailing whitespace here indeed, but don't forget to place a newline after this!